### PR TITLE
Add get_total_elapsed_time() to Tween

### LINKED ISSUE #4251: Add way of telling elapsed time to Tween
### Describe the project you are working on

A declarative UI library for godot 4

### Describe the problem or limitation you are having in your project

I want to carefully control tweening node properties,
whenever something changes I need to change the tween to accommodate the target and also the duration so that relative time spent stays the same.
 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add **back** the Tween::tell method that was present in godot 3.x as this provides the necessary info I need

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Have Tween return the elapsed time (which it undoubtedly already knows internally) when the method `tell` is called on it

### If this enhancement will not be used often, can it be worked around with a few lines of script?

There is no way to access this information otherwise

### Is there a reason why this should be core and not an add-on in the asset library?

Only the Tween class knows about it

### PR DESCRIPTION
**Pull request by KoBeWi**
Closes https://github.com/godotengine/godot-proposals/issues/4251

### CODE REVIEW

File: doc/classes/Tween.xml [hunk 1]
```diff
@@ -72,6 +72,13 @@
 				[b]Note:[/b] The [Tween] will become invalid after finished, but you can call [method stop] after the step, to keep it and reset.
 			</description>
 		</method>
+		<method name="get_total_elapsed_time" qualifiers="const">
+			<return type="float" />
+			<description>
+				Returns the total time in seconds the [Tween] has been animating (i.e. time since it started, not counting pauses etc.). The time is affected by [method set_speed_scale] and [method stop] will reset it to [code]0[/code].
+				[b]Note:[/code] Due to floating point precision, the time returned after the [Tween] has finished animating may slightly differ from the expected time.
```
Comment (KoBeWi) [inline doc/classes/Tween.xml:79]: The note is not perfectly accurate. The problem is basically that
```GDScript
var time = 0
func _process(delta):
    time += delta
    if time == 2:
        print("end")
```
will never be true due to delta precision. I wasn't sure how to word it.

Comment (akien-mga) [reply doc/classes/Tween.xml:79]: Did you fix this with #59571 ?

Comment (KoBeWi) [reply doc/classes/Tween.xml:79]: No, it's a different thing. The fix was about value, where the final is defined. Time doesn't really have a final value, it's just counted.

Comment (akien-mga) [reply doc/classes/Tween.xml:79]: Maybe phrase it like this:
```suggestion
				[b]Note:[/code] As it results from accumulating frame deltas, the time returned after the [Tween] has finished animating will be slightly greater than the actual [Tween] duration.
```

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**